pippy/PipelineSchedule.py

```diff
@@ -19,7 +19,7 @@
 
 class PipelineStage(ABC, nn.Module):
     @abstractmethod
-    def forward(self, args: List[torch.tensor]) -> torch.tensor:
+    def forward_one_chunk(self, args: List[torch.tensor]) -> torch.tensor:
```

Comment:
i dont have a strong preference, but is `forward_one_microbatch` a clearer name?

Reply:
Yeah, `chunk` is more from the old pippy implementation, like `split_args_kwargs_into_chunks`, `merge_chunks`, etc. I don't have a strong pref either, lol